### PR TITLE
feat(helprace): Add Helprace integration

### PR DESCRIPTION
**Pull request by Zerstoren**
We added support for use with Helprace. Our team has tested this internally and it is work.

### CODE REVIEW

**Comment by tcrammond** [comment]
Hello @Zerstoren, could you please check the contents of your PR again? It looks like there is something extra by accident. Please amend your branch to remove the extra file

**Comment by Zerstoren** [comment]
@tcrammond Thank you. I remove file.

**Comment by Zerstoren** [comment]
@tcrammond I deleted the file and issued it in one commit

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ out of the way and focus on real work.
 - [GQueues](https://www.gqueues.com/)
 - [Habitica](https://habitica.com)
 - [HEFLO](https://app.heflo.com/)
+- [Helprace](http://helprace.com/)
```
Comment (glensc) [inline README.md]: use https url

Comment (Zerstoren) [reply README.md]: Thanks. The problem is fixed.

**Comment by tcrammond** [comment]
I signed up for a new Helprace account and tried to use the integration, but it doesn't seem to be working for me. I'm on a page like `https://ACCOUNT.helprace.com/admin#tickets/view/t2?filterId=XXXXXXX`

Can you advise where it should be working? Do I need to enable any settings? I see that the selector looks for `.ticket_toggl` but I don't see in the page

**Comment by Zerstoren** [comment]
@tcrammond We have ready code, but which is not yet on the server for users. We are awaiting the adoption of a pull request to publish these changes. A feature of the work is toggl switch off. If we publish our changes before accepting the pull request, this will mislead our users. Is it possible to accept a pull request before the actual check?

**Comment by tcrammond** [comment]
This has been published to browsers in version `1.59.0`

Please let us know the feature toggle in Helprace is released!

**Pull request merged**